### PR TITLE
Changed the translation

### PR DESCRIPTION
**Pull request by Ruben80**
Did the following:

Rename "follow" (and similar occurrences) to "subscribe" in the code
Change the translations from "follow" and "folgen" to "subscribe" and "abonnieren"

The last one I did not know what to do: Rename the FOLLOW relationship in the database and write a migration to reflect that change

### Issues

- None

### Todo

- [X] None

### CODE REVIEW

File: webapp/locales/de.json [hunk 1]
```diff
@@ -351,8 +351,8 @@
     }
   },
   "followButton": {
-    "follow": "Folgen",
-    "following": "Folge Ich"
+    "follow": "Abonnieren",
+    "following": "Abonniere Mich"
```
Comment (Mogge) [inline webapp/locales/de.json]: This sounds strange. May bie just `Abonniert`?

File: webapp/locales/de.json [hunk 2]
```diff
@@ -498,9 +498,9 @@
   },
   "profile": {
     "commented": "Kommentiert",
-    "follow": "Folgen",
-    "followers": "Folgen",
-    "following": "Folge Ich",
+    "follow": "Abonnieren",
+    "followers": "Abonnieren",
```
Comment (Mogge) [inline webapp/locales/de.json]: Once again, this sounds strange. `Wird abonniert von` ?

File: webapp/locales/de.json [hunk 3]
```diff
@@ -498,9 +498,9 @@
...
+    "following": "Auf Abonniert",
```
Comment (Mogge) [inline webapp/locales/de.json]: `Hat abonniert` ?

File: webapp/locales/de.json [hunk 4]
```diff
@@ -510,10 +510,10 @@
...
     "network": {
       "andMore": "und {number} weitere …",
-      "followedBy": "wird gefolgt von:",
-      "followedByNobody": "wird von niemandem gefolgt.",
-      "following": "folgt:",
-      "followingNobody": "folgt niemandem.",
+      "followedBy": "wird abboniert von:",
+      "followedByNobody": "wird von niemandem abonniert.",
+      "following": "abonniert:",
+      "followingNobody": "abboniert auf niemandem.",
```
Comment (Mogge) [inline webapp/locales/de.json]: `abonniert niemanden`

**Comment by mattwr18** [comment]
closing this as we've been waiting more than a month for a response from @Ruben80 
and it's from a `fork` so there is no way for it to pass the build/get merged. See :point_up_2:

**Pull request closed without merging**